nurseconnect/tests/test_views.py

```diff
@@ -18,13 +18,17 @@ class MenuTestCase(MoloTestCaseMixin, TestCase):
     def setUp(self):
         self.mk_main()
         self.client = Client()
+        self.section = self.mk_section(self.section_index,
+                                       title="Section")
 
     def test_menu_renders_correctly(self):
         # Content in this view is dependent on there being sections.
         # So we'll test that the view returns successfully and leave
         # it at that
         response = self.client.get(reverse("menu"))
         self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response,
+                                "core/tags/section_listing_menu.html")
 
```

Comment:
You might want to assert for the two cases for the different scenarios, logged in and logged out.
assert for something obvious like "Login in response" and  "Login not in response"

Reply:
@moh-moola I have added them